webapp/lib/view.dart

```diff
@@ -581,55 +614,42 @@ class PackageConfiguratorView extends BaseView {
     var packageList = new Element.ul()
       ..classes.add('selected-active-package-list');
 
-    activePackages.forEach((packageName, selected) {
+    for (var package in activePackages) {
       packageList.append(
         new DivElement()
           ..classes.add('selected-active-package-list__item')
-          ..append(
-            new CheckboxInputElement()
-              ..classes.add('selected-active-package-list__item-state')
-              ..checked = selected
-          )
+          ..classes.toggle('selected-active-package-list__item--selected', (package == controller.selectedPackage))
           ..append(
             new Element.li()
               ..classes.add('selected-active-package-list__item-text')
-              ..text = packageName
+              ..text = package
           )
+          ..onClick.listen((_) => controller.command(controller.UIAction.loadPackageConfigurationView, new controller.PackageConfigurationData(package)))
       );
-    });
+    };
 
     _packageConfiguratorSidebar.append(packageList);
   }
-}
-
-class BatchRepliesConfigurationView extends PackageConfiguratorView {
-  DivElement hasAllTagsContainer;
-  DivElement containsLastInTurnTagsContainer;
-  DivElement hasNoneTagsContainer;
-  DivElement suggestedRepliesContainer;
-  DivElement addsTagsContainer;
-  BatchRepliesConfigurationView(model.Configuration data) : super() {
-    _buildContentPartial(data);
-  }
 
-  void _buildContentPartial(model.Configuration data) {
+  void _buildContentPartial() {
+    if (configurationData == null) return;
```

Comment:
Can we assume/enforce that it's never null and avoid this check? basically - try to avoid using `null` as a valid state. If you're just being cautious and it shouldn't happen that the config data is `null`, then you can just remove this line, otherwise let's look into when it's null and whether we can avoid calling this method unnecessarily.

Reply:
Removed the null check